src/server/core/libraries.processor.ts

```diff
@@ -9,13 +9,17 @@
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 
-export function processLibraries(libraries: string) {
+export function processLibraries(libraries: string, isMakerScript: boolean, isInsideOffice: boolean) {
```

Comment:
Now that there are three parameters (of which two are booleans), it might be good to make this into an option bag.